build/compile.mk

```diff
@@ -40,10 +40,21 @@ compile-linux: deps-only
 	@for b in $(BINS); do \
 		OUTPUT_FILE="$(BUILD_DIR)linux/$$b" ; \
 		echo "=== $(PROJECT_NAME) === [ compile-linux    ]:     $$OUTPUT_FILE"; \
-		BUILD_FILES=`find $(SRCDIR)/cmd/$$b -type f -name "*.go"` ; \
+		BUILD_FILES="$(SRCDIR)/cmd/..." ; \
 		GOOS=linux $(GO_CMD) build -ldflags="$(LDFLAGS)" -o $$OUTPUT_FILE $$BUILD_FILES ; \
 	done
 
+.PHONY: compile-linux-fips
+compile-linux-fips: deps-only
+	@echo "=== $(PROJECT_NAME) === [ compile-linux-fips    ]: building commands:"
+	@mkdir -p $(BUILD_DIR)/linux-fips
+	@for b in $(BINS); do \
+		OUTPUT_FILE="$(BUILD_DIR)linux-fips/$$b" ; \
+		echo "=== $(PROJECT_NAME) === [ compile-linux-fips    ]:     $$OUTPUT_FILE"; \
+		BUILD_FILES="$(SRCDIR)/cmd/..." ; \
+		GOOS=linux GOFIPS=1 CGO_ENABLED=1 CC=aarch64-linux-gnu-gcc GOEXPERIMENT=boringcrypto $(GO_CMD) build -tags fips -ldflags="$(LDFLAGS)" -o $$OUTPUT_FILE $$BUILD_FILES ; \
```

Comment:
I have not context on this but i assume that is correct, perhaps @alvarocabanas  could double check